### PR TITLE
Keyboard fixes

### PR DESCRIPTION
**Pull request by lilyfromseattle**
I addressed several tickets in this PR, all related to the keyboard and semi-dependent on each other. I also added tap to close functionality to the keyboard - if there is an existing ticket for this please let me know

https://user-images.githubusercontent.com/5643895/113346241-96409f80-9301-11eb-8be7-5aec026dbc38.mov

. Shoutout to @ashleyjelks for test help!

# PURCHASE-2516
Within initial inquiry view, keyboard is expanded (showing) by default

https://www.notion.so/artsy/Within-initial-inquiry-view-keyboard-is-expanded-showing-by-default-499d5576f5dc4f01aa794f0f0ce31d85

# PURCHASE-2522
When the keyboard is active can the make offer button hide? And when the keyboard is hidden the button is back.

https://www.notion.so/artsy/When-the-keyboard-is-active-can-the-make-offer-button-hide-And-when-the-keyboard-is-hidden-the-butt-5d2caef9172f4d86abde4e32dfca4975

### CODE REVIEW

File: src/lib/Scenes/Inbox/Components/Conversations/Composer.tsx [hunk 1]
```diff
@@ -59,6 +60,7 @@ export default class Composer extends React.Component<Props, State> {
     this.state = {
       active: false,
       text: null,
+      hideCTA: false,
```
Comment (erikdstock) [inline src/lib/Scenes/Inbox/Components/Conversations/Composer.tsx:63]: It seems this is not being used

File: src/lib/Scenes/Inbox/Screens/Conversation.tsx [hunk 2]
```diff
@@ -167,43 +167,45 @@ export class Conversation extends React.Component<Props, State> {
...
-          {!this.state.isConnected && <ConnectivityBanner />}
-          <Messages
-            componentRef={(messages) => (this.messages = messages)}
-            conversation={conversation as any}
-            onDataFetching={(loading) => {
-              this.setState({ fetchingData: loading })
-            }}
-          />
-        </Container>
+        <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}>
```
Comment (erikdstock) [inline src/lib/Scenes/Inbox/Screens/Conversation.tsx:170]: 🤯 I was afraid to pick up this ticket to be honest. glad it was straightforward

**Review by erikdstock — COMMENTED** [review]
One minor comment, looking pretty good here

File: src/lib/Scenes/Inbox/Components/Conversations/Composer.tsx (hunk 1 above)
Comment (lilyfromseattle) [reply src/lib/Scenes/Inbox/Components/Conversations/Composer.tsx:63]: Good catch!

**Review by erikdstock — APPROVED** [review]
🔥

**Pull request merged**